Proxy.h

```diff
@@ -1,20 +1,15 @@
 #pragma once
 
 #include "API.h"
-#include <mutex>
-
 namespace jni
 {
 
-class ProxyTracker;
-
 class ProxyObject : public virtual ProxyInvoker
 {
 // Dispatch invoke calls
 public:
 	virtual jobject __Invoke(jclass clazz, jmethodID mid, jobjectArray args);
-// Cleanup all proxy objects
-	static void DeleteAllProxies();
+	virtual void DisableProxy() = 0;
```

Comment:
can you add coverge for this new function ? can i call DisableProxy twice?

Reply:
Good one, it revealed that I misplaced it and it was protected :)